src/Language/PureScript/Parser/Kinds.hs

```diff
@@ -12,14 +12,25 @@ import Language.PureScript.Parser.Lexer
 import qualified Text.Parsec as P
 import qualified Text.Parsec.Expr as P
 
+parseStar :: TokenParser Kind
+parseStar = symbol' "*" *>
+  P.parserFail "The `*` symbol is no longer used for type kinds.\n  The new equivalent is the named kind `Type`."
```

Comment:
Would "type kinds" be better as "the kind of types" perhaps? (Same for effects below)